scripts/uploads/merge_and_uncode_rca_uploads.R

```diff
@@ -523,7 +487,16 @@ get_irr <- function(data) {
   return(irr)
 }
 
-save_to_synapse <- function(path, parent_id, file_name = NA, prov_name = NA, prov_desc = NA, prov_used = NA, prov_exec = NA) {
+#' Stores the file, version comment and provenance to Synapse
+#' @param path (string) name of cohort
+#' @param parent_id (string) whether we are running in production env or staging env
```

Comment:
I see your point that you want to flag that the parent_id should be the folder in either production or staging proejcts. But I think we should also point out that it should be a folder synapse id.

Reply:
Oops thanks for the catch. I think my comments got scrambled up.